docs/diagrams/ModelClassDiagram.puml

```diff
@@ -4,23 +4,28 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-Package Model as ModelPackage <<Rectangle>>{
-Class "<<interface>>\nReadOnlyAddressBook" as ReadOnlyAddressBook
-Class "<<interface>>\nReadOnlyUserPrefs" as ReadOnlyUserPrefs
-Class "<<interface>>\nModel" as Model
-Class AddressBook
-Class ModelManager
-Class UserPrefs
-
-Class UniquePersonList
-Class Person
-Class Address
-Class Email
-Class Name
-Class Phone
-Class Tag
-
-Class I #FFFFFF
+Package Model as ModelPackage <<Rectangle>> {
+    Class "<<interface>>\nReadOnlyAddressBook" as ReadOnlyAddressBook
+    Class "<<interface>>\nModel" as Model
+    Class ModelManager
+    Class "<<interface>>\nReadOnlyUserPrefs" as ReadOnlyUserPrefs
+    Class AddressBook
+    Class UserPrefs
+
+    Class UniquePersonList
+    Class UniqueWeddingList
+    Class Person
+    Class Wedding
+    Class Client
+    Class Address
+    Class Email
+    Class Name
+    Class Phone
+    Class Role
+    Class Date
+    Class Venue
+
+    Class I #FFFFFF
```

Comment:
![image](https://github.com/user-attachments/assets/3806c6d2-47f5-4dfc-b1ba-197e870ace57)
 client should point to person